designsafe/apps/rapid/views.py

```diff
@@ -68,6 +81,80 @@ def get_events(request):
     out = [h.to_dict() for h in s[0:total]]
     return JsonResponse(out, safe=False)
 
+@csrf_exempt
+@require_GET
+def opentopo_data(request):
+    url = 'https://portal.opentopography.org/API/otCatalog?productFormat=PointCloud&minx=-180&miny=-90&maxx=180&maxy=90&detail=true&outputFormat=json&include_federated=false'
+    
+    cache_key = f"proxy_response_{url}"
```

Comment:
Django has a cache_page decorator that I think can be used here instead of handling the cache keys manually. We're using it for Datacite requests: https://github.com/DesignSafe-CI/portal/blob/9dd9d563e0af7e0bb0c4380c433a66a6fb4aa426/designsafe/apps/api/publications/urls.py#L12

Reply:
Fixed with https://github.com/DesignSafe-CI/portal/pull/1417. thanks for the feedback.